spec/jobs/sample_accessioning_job_spec.rb

```diff
@@ -21,56 +21,78 @@
   end
 
   describe '#perform' do
-    context 'when the submission is successful' do
-      before do
-        allow(submission).to receive(:update_accession_number).and_return(true)
-      end
-
-      it 'does not raise an error' do
-        expect { job.perform }.not_to raise_error
-      end
+    before do
+      allow(described_class).to receive(:contact_user).and_return(contact_user)
```

Comment:
is the contact user that guy that left sanger years ago? is changing that to a PSD team email part of the changes you are doing here or a separate story (I forget)?